charts/orchestrator-k8s/values.yaml

```diff
@@ -96,6 +96,42 @@ backstage:
     ingress:
       enabled: true  # Use Kubernetes Ingress instead of OpenShift Route
     backstage:
+      extraVolumes:
+        - name: backstage-locations
+          configMap:
+            name: backstage-locations
+
+        - name: dynamic-plugins-root
```

Comment:
regarding lines 104-134 - since I override the volume mounts I had to re-add those mounts from the original janus-idp chart